en/docs/learn/examples/endpoint-examples/using-websocket-endpoints.md

```diff
@@ -2,75 +2,69 @@
 
 WebSocket is a protocol that provides full-duplex communication channels over a single TCP connection. This can be used by any client or server application. The Micro Integrator provides WebSocket support via the [WebSocket Transport]({{base_path}}/install-and-setup/setup/transport-configurations/configuring-transports/#configuring-the-websocket-transport) and the [WebSocket Inbound Protocol]({{base_path}}/learn/examples/inbound-endpoint-examples/inbound-endpoint-secured-websocket).
```

Comment:
```suggestion
WebSocket is a protocol that provides full-duplex communication channels over a single TCP connection. This can be used by any client or server application. The WSO2 Micro Integrator (MI) provides WebSocket support via the [WebSocket Transport]({{base_path}}/install-and-setup/setup/transport-configurations/configuring-transports/#configuring-the-websocket-transport) and the [WebSocket Inbound Protocol]({{base_path}}/learn/examples/inbound-endpoint-examples/inbound-endpoint-secured-websocket).
```